modules/ROOT/partials/configuration/exportword_watermark.adoc

```diff
@@ -0,0 +1,26 @@
+[[watermark]]
+== `watermark`
+
+The `watermark` option provides integrators with a way to include semi-transparent image watermarks on each page of the exported document, which can be useful for branding, copyright protection, or decorative purposes.
+
+*Type:* `+Object+`
+
+=== Example : using `watermark`
+
+[source,js]
+----
+tinymce.init({
+  selector: "textarea",
+  plugins: ['exportword'],
+  toolbar: 'exportword',
+  exportword_converter_options: {
+    watermark: {
+      source: 'http://moxiecode.cachefly.net/tinymce/v9/images/logo.png',
```

Comment:
Maybe not the best URL to use for a watermark. Could we change it to like `https://<your domain>/some-image.png` or something?

Reply:
```suggestion
      source: 'https://<your domain>/some-image.png',
```

Reply:
If its just a placeholder i think `https://<your domain>/some-image.png` will be suitable.